lib/msal-browser/docs/v4-migration.md

```diff
@@ -8,7 +8,34 @@ If you are coming from MSAL v4, you can follow this guide to update your code to
 
 ## API Breaking Changes
 
-[TBD]
+### TokenCache and loadExternalTokens
+
+The `TokenCache` object as well as the `getTokenCache` function in `PublicClientApplication` have been removed from MSAL.js.
+
+The `loadExternalTokens` API is now a separate export and requires a `Configuration` object in addition to the parameters accepted in MSAL.js v4. Please add the configuration object passed into your application to `loadExternalTokens`.
+
+```js
+// BEFORE
+
+const pca = new PublicClientApplication(config);
+await pca.getTokenCache().loadExternalTokens(
+    silentRequest,
+    serverResponse,
+    loadTokenOptions
+);
+
+//AFTER
+
+const pca = new PublicClientApplication(config);
```

Comment:
Nit: Just to make it clear PCA isn't required at all to use loadExternalTokens let's omit it here
```suggestion
```